core/src/main/java/org/apache/iceberg/rest/ResourcePaths.java

```diff
@@ -26,6 +26,56 @@
 public class ResourcePaths {
   private static final Joiner SLASH = Joiner.on("/").skipNulls();
   private static final String PREFIX = "prefix";
+  public static final String V1_NAMESPACES = "/v1/{prefix}/namespaces";
+  public static final String V1_NAMESPACE = "/v1/{prefix}/namespaces/{namespace}";
+  public static final String V1_NAMESPACE_PROPERTIES =
+      "/v1/{prefix}/namespaces/{namespace}/properties";
+  public static final String V1_TABLES = "/v1/{prefix}/namespaces/{namespace}/tables";
+  public static final String V1_TABLE = "/v1/{prefix}/namespaces/{namespace}/tables/{table}";
+  public static final String V1_TABLE_REGISTER = "/v1/{prefix}/namespaces/{namespace}/register";
+  public static final String V1_TABLE_METRICS =
+      "/v1/{prefix}/namespaces/{namespace}/tables/{table}/metrics";
+  public static final String V1_TABLE_RENAME = "/v1/{prefix}/tables/rename";
+  public static final String V1_TRANSACTIONS_COMMIT = "/v1/{prefix}/transactions/commit";
+  public static final String V1_VIEWS = "/v1/{prefix}/namespaces/{namespace}/views";
+  public static final String V1_VIEW = "/v1/{prefix}/namespaces/{namespace}/views/{view}";
+  public static final String V1_VIEW_RENAME = "/v1/{prefix}/views/rename";
+
+  // namespace endpoints
+  public static final Endpoint V1_LIST_NAMESPACES =
```

Comment:
I'm still undecided whether this is the right place for all of these endpoints, so I'd like to hear what others think

Reply:
yeah it would be either in `ResourcePaths` or in `Endpoint` itself

Reply:
I eventually moved stuff to `Endpoint` as that seems more appropriate